.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a bug report to help improve the ONNX.
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+# Bug Report
```

Comment:
Add a clause to confirm that this issue is not related to general model conversion. If model conversion is failing for a tutorial in this repo then we should accept the bug and fix the tutorial however if the bug is related to general model conversion then it should go to the appropriate converter repo